mesheryctl/pkg/utils/registry.go

```diff
@@ -212,3 +212,49 @@ func GenerateMDStyleDocs(model ModelCSV, components []ComponentCSV, modelPath, i
 
 	return nil
 }
+
+func GenerateIcons(model ModelCSV, components []ComponentCSV, imgPath string) error {
```

Comment:
Is this a copy from Meshery Server's code?

Does this come insert the necessary `<xml>` tag?

Reply:
`<?xml version="1.0" encoding="UTF-8"?><!DOCTYPE svg>`

Reference - https://docs.meshery.io/project/contributing/contributing-ui#static-files-icons-and-images

Reply:
Yes, it insert necessary xml tags, no it is not copy of server

Reply:
This code is present in mesheryctl however it was used in 3 places, so I exracted this code and made a generic function that can be reused instead of writing this code again and again

Reply:
I propose that it be a copy of Meshery Server... in terms of the treatment of SVGs.

Good of you to consolidate to single place in mesheryctl.

Reply:
Okay I'll check, last time Pranav worked on this and added this code, okay I'll check if he used the same code that we have in server or different

Reply:
Thanks. If it doesn’t make sense to copy / be the same, then please ignore this comment.